docs/README.md

```diff
@@ -0,0 +1,137 @@
+# docs
+
+The source for the documentation.
+
+## To Build
+
+To build:
+
+0. Prereq: [python3](https://www.python.org/downloads/)
```

Comment:
Nit -- now that the build expects api docs, we probably should update as a prereq to npm ci && npm run build-api-docs 

Although, I doubt anyone in the next 72 hours will try to build the docs 😄 , so if you want, I can update later.

Reply:
Thanks, fixed.